File: blog-sever/server-jpa/src/main/kotlin/com/site/blog/model/entity/BlogConfig.kt
package com.site.blog.model.entity


import com.fasterxml.jackson.annotation.JsonFormat
import jakarta.persistence.*
import java.time.LocalDateTime
 @Entity
data class BlogConfig  (


    /**
     * 字段名
     */
    var  configCode: String  = "",

    /**
     * 配置名
     */
    var  configName: String  = "",

    /**
     * 配置项的值
     */
    var  configValue: String? = null,

    /**
     * 创建时间
     */
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    var  createTime: LocalDateTime? = null,

    /**
     * 修改时间
     */
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    var  updateTime: LocalDateTime? = null) {
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
     @Column(name = "id", nullable = false)
     open var id: Long? = null
 }

     
 

File: BlogApi/BlogApi/Models/BlogConfig.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BlogApi.Models;

[Table("blog_config")]
public class BlogConfig
{
    [Key]
    [Column("id")]
    public string Id { get; set; } = Guid.NewGuid().ToString("N");

    [Column("config_code")]
    public string? ConfigCode { get; set; }

    [Column("config_name")]
    public string? ConfigName { get; set; }

    [Column("config_value")]
    public string? ConfigValue { get; set; }

    [Column("create_time")]
    public DateTime? CreateTime { get; set; }

    [Column("update_time")]
    public DateTime? UpdateTime { get; set; }
}
